File: BaekJoon/2908_상수.java
import java.util.*;
public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		String s1 = sc.nextLine();
		int a = Integer.parseInt(s1.charAt(2) + "" + s1.charAt(1) + "" +s1.charAt(0));
		int b = Integer.parseInt(s1.charAt(6) + "" + s1.charAt(5) + "" +s1.charAt(4));		
		if (a > b) System.out.print(a);
		if (a < b) System.out.print(b);
	}
}

File: BaekJoon/2908_상수.kt
import java.util.*
fun main (args: Array<String>) {
    var sc = Scanner(System.`in`)
    var a = sc.next()
    var b = sc.next()
    var save1 = ""
    for (i in a!!.length-1 downTo 0) {
        save1 += a.get(i)
    }
    var save2 = ""
    for (i in b!!.length-1 downTo 0) {
        save2 += b.get(i)
    }
    if (save1.toInt() > save2.toInt()) {
        print(save1)
    }
    else print(save2)
}
